src/main.js

```diff
@@ -1,3 +1,21 @@
 const core = require('@actions/core');
+const { action_library } = require('./action_library');
 
-core.info('Hello!');
+let library = new action_library();
+
+let scope = core.getInput('scope');
+let createOutputVariables = core.getInput('create-output-variables') == 'true';
+
+let inputFilePath = core.getInput('input-file');
+let inputYaml = library.getFileYaml(inputFilePath);
+
+let environmentYaml = library.getCurrentEnvironmentVars();
+let environmentDictionary = library.buildEnvironmentDictionary(scope, inputYaml, environmentYaml);
+
+console.log('Scoped Variables:', environmentDictionary);
+for (envVar in environmentDictionary) {
+  library.setEnvironmentVar(`${envVar}`, environmentDictionary[envVar]);
```

Comment:
Can you pass this without the template string (the method that uses it puts it in a template string too)?
```suggestion
  library.setEnvironmentVar(envVar, environmentDictionary[envVar]);
```

Reply:
I can take these out.  This was left over from some test I was doing with scenarios that included spaces and commas.